### PR TITLE
Make /app good for for 0.74

### PR DESCRIPTION
**Pull request by tjzel**
## Summary

This PR is a part of ongoing effort to bring Reanimated to React Native 0.74 with bridgeless mode. It consists of following Pull Requests, which are split for review convenience - but shouldn't be considered separate entities. They have to be merged in the following order:

- #5830
- #5815
- #5834
- #5734 

## Test plan

:shipit:

### CODE REVIEW

File: app/src/examples/AboutExample.tsx
Comment (tjzel) [reply app/src/examples/AboutExample.tsx:73]: I did to print it in `AboutExample` 🤣. We can remove it if we don't want it.

Comment (piaskowyk) [reply app/src/examples/AboutExample.tsx:73]: I'm ok with that 😅

**Review by piaskowyk — APPROVED** [review]
For me is legit 🫡  Before merging:
- Let's sort out the CI - https://github.com/software-mansion/react-native-reanimated/actions/runs/8421856626/job/23059739159?pr=5830
- Wait for approval on https://github.com/software-mansion/react-native-reanimated/pull/5815, and then let's merge both pull requests together to keep the main branch in a consistent state.

File: app/package.json
Comment (tjzel) [reply app/package.json]: I have them all upgraded in other PRs 👍

File: app/src/examples/LiquidSwipe/LiquidSwipe.tsx
Comment (tomekzaw) [inline app/src/examples/LiquidSwipe/LiquidSwipe.tsx:38]: remove `ctx`

File: app/src/examples/AboutExample.tsx
Comment (tomekzaw) [inline app/src/examples/AboutExample.tsx:73]: move to `isBridgeless()`

File: app/src/examples/ScreenStackHeaderConfigBackgroundColorExample.tsx
Comment (tomekzaw) [inline app/src/examples/ScreenStackHeaderConfigBackgroundColorExample.tsx:24]: 😢

**Review by tomekzaw — CHANGES_REQUESTED** [review]

**Review by tomekzaw — APPROVED** [review]

**Pull request merged**
